systers_portal/systers_portal/settings/dev.py

```diff
@@ -1,23 +1,24 @@
 from .base import *
+from decouple import config
 
-DEBUG = True
+DEBUG = config('DEBUG', default=False, cast=bool)
 TEMPLATES[0]['OPTIONS']['debug'] = DEBUG
 
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'systersdb',
-        'USER': '',
-        'PASSWORD': '',
-        'HOST': 'localhost',
-        'PORT': '5432',
+        'NAME': config('DB_NAME'),
```

Comment:
Let's keep the original values as default values here for the sake of sanity when there is no `.env` file.

Reply:
You want me to remove the config() and keep it as it was in dev.py for all the variables (NAME, USER, PASSWORD, HOST, PORT, EMAIL_HOST, EMAIL_PORT) ?

Reply:
No, there is a way to specify default values
```
config('EMAIL_PORT', default=25, cast=int)
```

Just keep the original values as default values.

Reply:
Oh, OK Ok